statsmodels/graphics/tsaplots.py

```diff
@@ -186,9 +203,17 @@ def plot_pacf(x, ax=None, lags=None, alpha=.05, method='ywunbiased',
     Adapted from matplotlib's `xcorr`.
 
     Data are plotted as ``plot(lags, corr, **kwargs)``
+
+    kwargs is used to pass properties to both the line for the
+    autocorrelations and for the horizontal line at 0. These properties must
+    be valid for a Line2D object.
+
+    vlines_kwargs is used to pass additional properties to the vertical
+    lines.  The keywords in this dictionary must be valid propertied for a
```

Comment:
typo?
propertied -> properties

however, I think using "options" instead of "properties" would be clearer. i.e. it's keyword arguments to the corresponding matplotlib function/method

Reply:
Swapped to options.